doc/visual-programming/source/widgets/evaluate/calibrationplot.md

```diff
@@ -7,21 +7,41 @@ Shows the match between classifiers' probability predictions and actual class pr
 
 - Evaluation Results: results of testing classification algorithms
 
-The [Calibration Plot](https://en.wikipedia.org/wiki/Calibration_curve)plots class probabilities against those predicted by the classifier(s).
+**Outputs**
 
-![](images/CalibrationPlot-stamped.png)
+- Calibrated Model: a model, calibrated with a user-set probability threshold
+
+The [Calibration Plot](https://en.wikipedia.org/wiki/Calibration_curve) plots class probabilities against those predicted by the classifier(s). We would use this widget to see whether a classifier is overly optimistic (gives predominantly positive results) or pessimistic (gives predominantly negative results). The widget can also output a calibrated model, where the user sets his/her own probability threshold.
+
```

Comment:
I would turn it around: it "plots probabilities predicted by the classifier against actual class probabilities", to have x vs. y and not vice versa.

Reply:
It is not overly optimistic vs. overly pessimistic. It can be both, or something completely different. I'd just remove this sentence.

Reply:
I would remove the last sentence and explain the outputs later (see below). I would, however, add a paragraph here:

The widget also allows the user to change the classification threshold from the default value of 0.5 to another value chosen according to criteria such as desired balance between precision and recall.